core/js/typespecs/common_types.ts

```diff
@@ -32,6 +33,7 @@ export const objectTypesWithEvent = z.enum([
   "experiment",
   "dataset",
   "prompt",
+  "prompt_session",
```

Comment:
I think this will cause us to generate REST API endpoints for prompt session from https://github.com/braintrustdata/braintrust/blob/b1e0672c1dc1200a0031e997eb908102c61b65be/deployment-internal/openapi/src/api_spec.ts#L727. Not sure if that is intended.

Reply:
We do eventually want to have those, but I think I needed to do this to wire things up so I could put `prompt_session` here to begin with. What do you suggest in the short term?

Reply:
I would just exclude "prompt_session" from that loop in the code I linked.